frontend/src/features/admin/api/createVendor.ts

```diff
@@ -49,6 +50,19 @@ export const createVendor = async (
 
     await supabase.rpc("update_vendor_location", { vendor_id: data.id });
 
+    // Add tags to the vendor
+    if (skillSouthAsian) {
```

Comment:
can you make this more generic? I'm fine with using another db call to get the tag id since this is an admin action